src/Application/Actions/Person/Create.php

```diff
@@ -6,10 +6,11 @@
 
 use BigGive\Identity\Application\Actions\Action;
 use BigGive\Identity\Application\Auth\Token;
+use BigGive\Identity\Client\BadRequestException;
+use BigGive\Identity\Client\Mailer;
 use BigGive\Identity\Domain\Person;
 use BigGive\Identity\Repository\PersonRepository;
 use Laminas\Diactoros\Response\TextResponse;
-use OpenApi\Annotations as OA;
```

Comment:
Removing this adds a warning in my IDE because the documentation metadata can no longer be looked up.